packages/migration/src/tests/mods/7.0.0/__snapshots__/navPackageMove.test.ts.snap

```diff
@@ -0,0 +1,31 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Nav package move moves import statements 1`] = `
+"import {  } from 'office-ui-fabric-react'
+import { Nav, INavLink } from '@uifabric/legacy'
+
+import {  } from 'office-ui-fabric-react'
+import { Nav } from '@uifabric/legacy'
+import {  } from 'office-ui-fabric-react'
+import { Nav } from '@uifabric/legacy'
```

Comment:
Why is this duplicated?

Reply:
Was attempting to validate multiple quote styles, but prettier "fixed" things for me after I ran commit. ☹️